src/clients/base-browser-client.js

```diff
@@ -98,7 +98,7 @@ class Client extends BaseClient {
     }
   }
 
-  __buildUrl (path, parameters) {
+  buildUrl (path, parameters) {
```

Comment:
I'm 100% sure this renaming will explode. 💥 
If you don't like the underscore `__buildUrl` function, then replace it in the whole `src/folder`

Reply:
Where do you see the "explosion"?

Reply:
It may be not an explosion, but it's definitely confusing for every who looks at it.

`base-browser-client.js` now defines a function `__buildUrl`, which is used by `nick-client.js`.
`base-browser-client.js` still uses `__buildUrl`, which works because it's inherited from `base-client.js`, which also defines a `__buildUrl`.

I think that's not a good solution and would suggest 
a) you replace `__buildUrl` everywhere with `buildUrl` or 
b) move `buildUrl` to `nick-client.js`.